src/main/scala/mesosphere/marathon/api/akkahttp/v2/EventsController.scala

```diff
@@ -52,15 +52,19 @@ class EventsController(
     authenticated.apply { implicit identity =>
       authorized(ViewResource, AuthorizedResource.Events).apply {
         parameters('event_type.*) { events =>
-          extractClientIP { clientIp =>
-            complete {
-              Source
-                .fromGraph[MarathonEvent, NotUsed](
-                  new EventStreamSourceGraph(eventBus, conf.eventStreamMaxOutstandingMessages(), clientIp)
-                )
-                .filter(isAllowed(events.toSet))
-                .map(event => ServerSentEvent(`type` = event.eventType, data = Json.stringify(Formats.eventToJson(event))))
-                .keepAlive(5.second, () => ServerSentEvent.heartbeat)
+          parameter("plan-format".?) { light =>
```

Comment:
Will something like this work here? 
```scala
(parameters("plan-format".?, 'event_type.*) & extractClientIP) { (light, events, clientIp) =>
```

Reply:
True, I forgot that `parameters` can accept more than one argument :smile:

Reply:
But I think the `extractClientIP` type is incompatible and cannot be conjugated.

Reply:
in AppsContoller we have something like `(entity(as[AppDefinition]) & forceParameter & extractClientIP & extractUri) { (rawApp, force, remoteAddr, reqUri) =>`, different types shouldn't be a problem :)

Reply:
Ah, true :smile: (forgot the parenthesis)